lib/xml/parser.js

```diff
@@ -39,7 +39,7 @@ function parseEventsMultistatus(xml) {
             // fix etag string (renders as '"[...]"', ugly xml2js objects (pew pew)
             let etag = event["d:propstat"][0]["d:prop"][0]["d:getetag"][0];
 
-            etag = etag.substring(1, etag.length);
+            etag = etag.substring(1, etag.length - 1);
```

Comment:
Btw: I never use `substr` and `substring`. `.slice()` does almost always that what you want 😁. In this case: `etag.slice(1, -1)` would be sufficient.